server/kolide/queue.go

```diff
@@ -0,0 +1,10 @@
+package kolide
+
+import (
+	"context"
+	"encoding/json"
+)
+
+type QueueService interface {
```

Comment:
Thought this was a simple interface for results and status and seams correct and would work for just about anything.